ops/ansible/playbooks-ccs/build_bfd-platinum.yml

```diff
@@ -22,4 +22,6 @@
           - java-11-openjdk-devel
           - git
           - ansible
+          - python2-boto3
+          - python2-botocore
```

Comment:
These new platinum dependencies are unfortunately required for the system level `aws_ssm` lookup plugin in ansible to work.